talk-proposals/SGibson-CogX20-BinderHub.md

```diff
@@ -0,0 +1,111 @@
+
+During the Covid-19 pandemic, mybinder.org has seen a significant increase in usage correlating with the beginning of lockdown and many researchers began working and collaborating remotely.
+It is quickly becoming the "go to" tool for sharing results and facilitating remote teaching in this new landscape of research culture.
+
+Dr Sarah Gibson is a member of the mybinder.org operating team and is personally responsible for maintaining the Binder infrastructure deployed at the Turing.
+During this talk, She will introduce Binder (the service), BinderHub (the technological infrastructure) and mybinder.org (a public instance of a Binder service, free for anyone to use) and demonstrate how it can be utilised to share reproducible computational environments and analyses.
+
+## Descriptors
+
+_Please select all descriptors that best apply to your talk._
+
+- [ ] Technical / specialist
+- [x] Big picture
+- [x] Detailed
+- [ ] Humorous
+- [ ] Serious
+- [ ] Retrospective
+- [x] Prospective
+- [ ] General / accessible
+
+## Domain Application
+
+_Please select up to two domains that best apply to your research and talk proposal._
+
+- [ ] Natural world
+- [ ] Security
+- [x] Engineering
+- [ ] Health
+- [x] Society & Culture
+- [ ] Economics
+
+## Methodological Application
+
+_Please select which methodological problem areas apply to your research and talk._
+
+- [ ] Finding ways to automate the lengthy process spent preparing data for analysis
+- [ ] Preventing misuse of data-driven technologies and artificial intelligence
+- [ ] Using and analysing data in order to confidently infer causal relationships
+- [x] Empowering citizens to access, understand and exploit the world's data
+- [ ] Improving the ways in which we collect and analyse data
+- [x] Effectively and efficiently convey information to varying audiences
+- [ ] Merging multiple sources of data
+- [ ] Using data while protecting privacy
+- [ ] Merging human knowledge with data-driven machine learning and AI systems
+- [ ] Eradicating failures within interacting systems
+- [ ] Optimising algorithmic and system performance for bigger and more complex datasets
+- [ ] Building on the most effective methods across data science to produce the next gen of techniques
+
+## Relevance
+
+_Please explain the relevance of your talk to the CogX audience, and the global impact of your research._
+_500 words max._
+
+In order to "get the next 10 years right", the research community needs to undergo a cultural and practical shift in how we share and communicate scientific progress, towards embracing openness and collaboration more than ever before.
+One part of this shift is embedding reproducibility into our scientific workflows.
+If code and data are not available for interrogation and results can't be reproduced, it becomes much more difficult to build on top of them and advance the domain.
+It is also unethical to do work that is closed and cannot be built upon quickly.
```

Comment:
I'd like to say more about _why_ it is unethical. But. The only examples I can think of are like the Health and Policy domains having a negative impact of people's quality of life. However, I think feel like I should caveat that with "whilst protecting people's privacy" and I feel like I'm now arguing against myself 😅

Reply:
@KirstieJane can you suggest a sentence here please?

Reply:
Actually never mind, I restructured this section 😄